waiter/src/waiter/service_description.clj

```diff
@@ -645,6 +645,8 @@
   (defn fetch-core
     "Loads the service description for the specified service-id from the key-value store."
     [kv-store ^String service-id & {:keys [refresh nil-on-missing?] :or {refresh false nil-on-missing? true}}]
+    (when refresh
+      (log/info "force fetching core service description for" service-id))
```

Comment:
👍 I like that this logging call was moved here from `service-id->service-description`.